### PR TITLE
mailmap updates for v11.1.0

### PR DESCRIPTION
**Pull request by theanalyst**

### CODE REVIEW

File: .organizationmap [hunk 1]
```diff
@@ -510,6 +510,7 @@ Unaffiliated <no@organization.net> Hong Zhangoole <hong.zhangoole@gmail.com>
 Unaffiliated <no@organization.net> Gaël Fenet-Garde <gael.fenet.garde@gmail.com>
 Unaffiliated <no@organization.net> Gaurav Kumar Garg <garg.gaurav52@gmail.com>
 Unaffiliated <no@organization.net> Germain Chipaux <germain.chipaux@gmail.com>
+Unaffiliated <no@organization.net> gcharot <buixor@gmail.com>
```
Comment (dachary) [inline .organizationmap]: Grégory Charot <gcharot@redhat.com>

File: .mailmap [hunk 2]
```diff
@@ -405,15 +419,20 @@ Yehuda Sadeh <yehuda@inktank.com> <yehuda@yehuda.infit.com>
...
 Yongqiang He <he.yongqiang@h3c.com>
 Yongyue Sun <abioy.sun@gmail.com>
 You Ji <youji@ebay.com>
 Yuan Zhou <yuan.zhou@intel.com> <dunk007@gmail.com>
 Yunchuan Wen <yunchuan.wen@kylin-cloud.com> <yunchuanwen@ubuntukylin.com>
 Zengran Zhang <zhangzengran@h3c.com>
 Zeqiang Zhuang <zhuang.zeqiang@h3c.com>
+Zhang Shaowen <zhang_shaowen@139.com>
 Zhang Zezhu <zhang.zezhu@zte.com.cn>
+Zhangdong Guo <18896723979@139.com>
```
Comment (idealguo) [inline .mailmap]: Guo Zhandong <guozhandong@cmss.chinamobile.com> <idealguo@139.com>, thanks

File: .organizationmap [hunk 3]
```diff
@@ -480,6 +480,7 @@ Unaffiliated <no@organization.net> Daniel Schepler <dschepler@gmail.com>
 Unaffiliated <no@organization.net> Darrell Enns <darrell@darrellenns.com>
 Unaffiliated <no@organization.net> David Anderson <dave@natulte.net>
 Unaffiliated <no@organization.net> Dennis Schafroth <dennis@schafroth.com>
+Unaffiliated <no@organization.net> DesmondS <jordanjimmy41177@gmail.com>
```
Comment (dachary) [inline .organizationmap]: 施柏安 Desmond Shih
>> 技術研發部 Technical Development
>>  <http://www.inwinstack.com/>
>> 迎棧科技股份有限公司
>> │ 886-975-857-982
>> │ desmond.s at inwinstack <desmond.s at inwinstack.com>
>> │ 886-2-7738-2858 #7441
>> │ 新北市220板橋區遠東路3號5樓C室
>> Rm.C, 5F., No.3, Yuandong Rd.,
>> Banqiao Dist., New Taipei City 220, Taiwan (R.O.C)

File: .organizationmap [hunk 4]
```diff
@@ -47,6 +48,9 @@ CERN <contact@cern.ch> Dan van der Ster <daniel.vanderster@cern.ch>
 CERN <contact@cern.ch> Hervé Rousseau <hroussea@cern.ch>
 CERN <contact@cern.ch> Joaquim Rocha <joaquim.rocha@cern.ch>
 CERN <contact@cern.ch> Sebastien Ponce <sebastien.ponce@cern.ch>
+China Mobile <contact@chinamobile.com> Gui Hecheng <guihecheng@cmss.chinamobile.com>
+China Mobile <contact@chinamobile.com> Jing Wenjun <jingwenjun@cmss.chinamobile.com>
+China Mobile <contact@chinamobile.com> Zhang Shaowen <zhangshaowen@cmss.chinamobile.com>
```
Comment (idealguo) [inline .organizationmap]: add China Mobile <contact@chinamobile.com> Guo Zhandong <guozhandong@cmss.chinamobile.com>, thanks

File: .organizationmap [hunk 5]
```diff
@@ -600,12 +625,14 @@ Unaffiliated <no@organization.net> Yatin Kumbhare <yatinkumbhare@gmail.com>
 Unaffiliated <no@organization.net> YiQiang Chen <cyqsign@163.com>
 Unaffiliated <no@organization.net> Yongyue Sun <abioy.sun@gmail.com>
 Unaffiliated <no@organization.net> You Ji <jiyou09@gmail.com>
+Unaffiliated <no@organization.net> Zhangdong Guo <18896723979@139.com>
```
Comment (idealguo) [inline .organizationmap]: Guo Zhandong <idealguo@139.com>, thanks

File: .organizationmap (hunk 4 above)
Comment (theanalyst) [inline .organizationmap]: thanks, will update

File: .organizationmap (hunk 1 above)
Comment (theanalyst) [inline .organizationmap]: updating

File: .organizationmap (hunk 5 above)
Comment (theanalyst) [inline .organizationmap]: ack

**Comment by theanalyst** [comment]
@idealguo updated the changes

File: .mailmap [hunk 6]
```diff
@@ -369,6 +385,7 @@ Wei Luo <weilluo@tencent.com>
 Wei Qian <weiq@dtdream.com>
 Wenjun Huang <wenjunhuang@tencent.com>
 Wido den Hollander <wido@42on.com> <wido@widodh.nl>
+Whu Liuchang <nfcj000@qq.com>
```
Comment (Liuchang0812) [inline .mailmap]: Chang Liu <liuchang0812@gmail.com> <nfcj000@qq.com>

thanks!

**Comment by theanalyst** [comment]
@dachary added updates wrt 11.2.0 release, there are some old inktank accounts coming up because of qa suite merge

**Comment by theanalyst** [comment]
@dachary have a couple of updates to do (after mailing authors), will wait for a couple of days before doing a rebase and then merge

**Comment by dachary** [comment]
ack

**Comment by theanalyst** [comment]
@dachary updated all the author comments, I guess this is ready for merge now

**Pull request merged**